tools/clang/lib/SPIRV/InstBuilderAuto.cpp

```diff
@@ -28,17 +29,43 @@ inline bool bitEnumContains(spv::MemoryAccessMask bits,
                             spv::MemoryAccessMask bit) {
   return (uint32_t(bits) & uint32_t(bit)) != 0;
 }
+} // namespace
```

Comment:
It seems this is just how clang closes anonymous namespace, see [here](https://github.com/google/DirectXShaderCompiler/blob/master/tools/clang/lib/CodeGen/MicrosoftCXXABI.cpp#L3337). Actually this comment is automatically added by clang-format. :)